### PR TITLE
Condense README sections

### PR DESCRIPTION
**Pull request by Bubblemelon**
Related PR #671 

See @hoegaarden 's [suggestions](https://github.com/kubernetes/release/pull/671#issuecomment-468222992).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,91 +19,24 @@ There are several scripts and helpers in this repository a release
...
+Tools | Description
+ :---: | --
+[`gcbmgr`](https://github.com/kubernetes/release/blob/master/gcbmgr) | Google Cloud Builder Manager: <br/><br/> This is the main entry point for release managers for producing releases. All release types can be staged and later released using this method.
+[`anago`](https://github.com/kubernetes/release/blob/master/anago) | Release Tool: <br/><br/> The main driver for creating staged builds and releases. This is what runs inside GCB after a job is submitted using `gcbmgr`.
+[`branchff`](https://github.com/kubernetes/release/blob/master/branchff) | Fast-forward branching helper : <br/><br/> A tool used to pull new patches onto the release branch.
+<br/> [`find_green_build`](https://github.com/kubernetes/release/blob/master/find_green_build) <br/><br/> | Asks Jenkins for a good build to use.
+<br/> [`release-notes`](https://github.com/kubernetes/release/blob/master/cmd/release-notes) <br/><br/> | Scrape GitHub for release notes. See [Release Notes Gathering](#release-notes-gathering) for more information.
+<br/> [`prin`](https://github.com/kubernetes/release/blob/master/prin) <br/><br/> | What tags/releases is my PR IN?
```
Comment (Bubblemelon) [inline README.md]: This was verbosely copied over from the old README. Does this simply mean, "_To show tags/releases that a PR is in_."  ?

File: README.md [hunk 2]
```diff
@@ -27,91 +19,24 @@ There are several scripts and helpers in this repository a release
...
+<br/> [`changelog-update`](https://github.com/kubernetes/release/blob/master/changelog-update) <br/><br/> | Updates [CHANGELOG.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md) version entries by rescanning github for text and label changes.
+<br/> [`push-build.sh`](https://github.com/kubernetes/release/blob/master/push-build.sh) <br/><br/> | Pushes a developer (or CI) build up to GCS.
```
Comment (Bubblemelon) [inline README.md]: This one was also verbosely copied over, does CI mean the CI-Signal team?

File: README.md (hunk 1 above)
Comment (hoegaarden) [reply README.md]: Yes.

E.g. when I am in the directory of the k8s source and call the script with a random PR number or a random commit hash, it shows which tags those PR/commit is included in. However, I have never ever used that script before.

```
user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$ /src/prin '69500'
Skipping security_layer::hosted_map extensions...

COMMITS: 3f102c3980 (initial)  (cherrypicks)
(https://github.com/kubernetes/kubernetes/pull/69500)

                LATENCY TO APPEAR (DAYS)
TAG             K8S
---             ---
v1.13.0         49.22
v1.13.0-alpha.1 3.10
v1.13.0-alpha.2 11.62
v1.13.0-alpha.3 18.57
v1.13.0-beta.0  26.01
v1.13.0-beta.1  35.42
v1.13.0-beta.2  41.26
v1.13.0-rc.1    46.32
v1.13.0-rc.2    47.36
v1.13.1         60.78
v1.13.1-beta.0  52.10
v1.13.2         88.56
v1.13.2-beta.0  61.67
v1.13.3         111.97
v1.13.3-beta.0  90.21
v1.13.4         137.53
v1.13.4-beta.0  112.06
v1.13.5-beta.0  138.79
v1.14.0-alpha.0 25.94
v1.14.0-alpha.1 94.68
v1.14.0-alpha.2 108.62
v1.14.0-alpha.3 122.58
v1.14.0-beta.0  129.81
v1.14.0-beta.1  136.64
v1.15.0-alpha.0 129.77

user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$ /src/prin '3f102c3980'
Skipping security_layer::hosted_map extensions...

COMMITS: 3f102c3980 (initial)  (cherrypicks)

                LATENCY TO APPEAR (DAYS)
TAG             K8S
---             ---
v1.13.0         49.22
v1.13.0-alpha.1 3.10
v1.13.0-alpha.2 11.62
v1.13.0-alpha.3 18.57
v1.13.0-beta.0  26.01
v1.13.0-beta.1  35.42
v1.13.0-beta.2  41.26
v1.13.0-rc.1    46.32
v1.13.0-rc.2    47.36
v1.13.1         60.78
v1.13.1-beta.0  52.10
v1.13.2         88.56
v1.13.2-beta.0  61.67
v1.13.3         111.97
v1.13.3-beta.0  90.21
v1.13.4         137.53
v1.13.4-beta.0  112.06
v1.13.5-beta.0  138.79
v1.14.0-alpha.0 25.94
v1.14.0-alpha.1 94.68
v1.14.0-alpha.2 108.62
v1.14.0-alpha.3 122.58
v1.14.0-beta.0  129.81
v1.14.0-beta.1  136.64
v1.15.0-alpha.0 129.77

user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$

```

File: README.md (hunk 2 above)
Comment (hoegaarden) [reply README.md]: "CI" in this context probably means "the CI system". In [the description of the script][desc] they e.g. talk about jenkins, which as far as I know was the precursor of [prow]. I have never used that script and consider it outdated.

[desc]: https://github.com/kubernetes/release/blob/9fe1ffac23fd77cc0459ff5d156fa2c526125587/push-build.sh#L32-L41
[prow]: https://git.k8s.io/test-infra/prow

File: README.md [hunk 3]
```diff
@@ -27,91 +19,24 @@ There are several scripts and helpers in this repository a release
...
+<br/> [`script-template`](https://github.com/kubernetes/release/blob/master/script-template) <br/><br/> | Generates a script template in the kubernetes/release ecosystem.
+
+For information on how to use `gcbmgr`, `anago` and `branchoff`, see the [Branch Manager Handbook](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/branch-manager#branch-manager-handbook)
```
[the remark this answers is not in the document]
Comment (Bubblemelon) [reply README.md]: Ohh typo!  Thanks for catching that! :blush:

File: README.md (hunk 2 above)
Comment (Bubblemelon) [reply README.md]: Oh okay, I get it now. I completely misinterpreted what that sentence meant,  looking at `push-build.sh` when the sentence says, "developer" it actually means developer build and CI for Jenkins continuous builds ( exactly what you've pointed out :sweat_smile: ).

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (hoegaarden) [reply README.md]: If you'd quickly fix that (either by accepting this suggestion or pushing a commit which fixes that) we can get this whole PR merged. Let me know, if you need help!

[the remark this answers is not in the document]
Comment (Bubblemelon) [reply README.md]: done! :sweat_smile:

**Comment by hoegaarden** [comment]
/lgtm
/ok-to-test

/hold
... for a bit, to maybe get another pair of eyes on it.
/assign @dougm @tpepper

**Comment by hoegaarden** [comment]
Thanks @dougm ... will probably check on deprecating some tools in a separate PR.

/hold cancel

**Pull request merged**